### ISSUE TITLE
Cant Stream data to Unreal

### ISSUE THREAD

**Issue by SquishySquid21** [fweidner/UE4-Plugin-OffAxis#13]
Hello,

First of all i apologies for my lack of proper english, second im trying to connect to unreal via optitrack but after calibration in optitrack, and using one market to send the position, nothing happens, i have three cameras for tracking and a 4 markers device for the rigidbody, what am i doing wrong? in streaming pane in motive i activate the broadcast on, and in unreal i have the project of OffAxisTest open but nothing happens, the position doesnt seem to be affected.

If someone need more descriptions of my project i can post screenshots with descriptions

Thank You

**Comment by fweidner** [comment]
Hi,

currently, tracking only works with rigid bodys. 
You need to create a rigid body in Motive and give it a name. In the OffAxisActor, you can insert the same name in the section "Tracking Optitrack" at "Head Name" (default: Head). In other words, Motive needs to track a Rigid Body called Head.

Best, 
Florian

**Comment by fweidner** [comment]
Hi,
1)
Currently, the OffAxisPlugin only accepts RigidBody data as input. 
In Motive, you need to have a RigidBody with a name (https://v20.wiki.optitrack.com/index.php?title=Rigid_Body_pane).
In the OffAxisActor in Unreal there is a section called "Tracking Optitrack" and there a variable called "Head Name" with default value "Head". This is the name of the rigid body used for updating the position. 
In other words, your Rigid Body in Motive needs to have the same name as the variable "Head Name". For example: "Head".

Also, you need to press "4" to connect to Optitrack. Please see the Input section of the Project Settings for all controls.

**Comment by SquishySquid21** [comment]
Thanks for the reply,

I cant find the section called "Tracking Optitrack", i prove the tracking cameras in a new project and it works to move a chair in the scene, but in that project i put optitrack client origin and rigidbody, i need to add that in this project too?(Client origin and Rigidbody).

**Comment by fweidner** [comment]
Are you using the original Optitrack Plugin or the one implemented by me?

**Comment by fweidner** [comment]
I've never tried it with both plugins enabled. This could be an issue. It should definitively work with my plugin. If you use the native Optitrack plugin, there might be some trouble with the coordinate systems (not sure, tho)
